src/registryctl/client/client.go

```diff
@@ -25,15 +26,24 @@ import (
 	"github.com/goharbor/harbor/src/common/http/modifier/auth"
 	"github.com/goharbor/harbor/src/common/utils"
 	"github.com/goharbor/harbor/src/lib/log"
-	"github.com/goharbor/harbor/src/registryctl/api"
+	"github.com/goharbor/harbor/src/registryctl/api/registry/gc"
+)
+
+// const definition
```

Comment:
This is the client core used to call registryctl's API right?
We should attach request ID and extract request ID on the server side.  
If you don't want to handle it in this PR could you open an issue?

Reply:
I will raise an issue for it, and handle it in another PR.

Reply:
ok